webapp/package.json

```
@@ -59,6 +60,11 @@
     "react-app-rewired": "^2.1.5",
     "typescript": "^4.5.2"
   },
+  "jest": {
+    "coverageReporters": [
+      "text-summary"
```

I think the complete `"text"` reporter is super interesting, with a table breakdown of the test coverage per file, including a row for "All files" with the same numbers as in `"text-summary"`. Can we change it?
```suggestion
      "text"
```

---

sure